modules/apps/commerce/commerce-product-content-search-web/src/main/resources/META-INF/resources/sort/configuration.jsp

```diff
@@ -0,0 +1,47 @@
+<%--
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+--%>
+
+<%@ include file="/init.jsp" %>
+
+<%
+CPSortDisplayContext cpSortDisplayContext = (CPSortDisplayContext)request.getAttribute(WebKeys.PORTLET_DISPLAY_CONTEXT);
+%>
+
+<liferay-portlet:actionURL portletConfiguration="<%= true %>" var="configurationActionURL" />
+
+<liferay-portlet:renderURL portletConfiguration="<%= true %>" var="configurationRenderURL" />
+
+<aui:form action="<%= configurationActionURL %>" method="post" name="fm">
+	<aui:input name="<%= Constants.CMD %>" type="hidden" value="<%= Constants.UPDATE %>" />
+	<aui:input name="redirect" type="hidden" value="<%= configurationRenderURL %>" />
+
+	<div class="portlet-configuration-body-content">
+		<div class="container-fluid container-fluid-max-xl">
+			<div class="sheet">
+				<div class="panel-group panel-group-flush">
+					<aui:select id="selectDefaultSort" label="select-default-sort" name="preferences--selectDefaultSort--" value="<%= cpSortDisplayContext.selectedDefaultSort() %>">
+
+						<%
+						for (String sortType : CPSearchResultsConstants.SORT_OPTIONS) {
+						%>
+
+							<aui:option label="<%= sortType %>" selected="<%= cpSortDisplayContext.selectedDefaultSort().equalsIgnoreCase(sortType) %>" />
+
```

Comment:
please save `cpSortDisplayContext.selectedDefaultSort()` in a variable before the for cycle (it should be the same for all the loop - and to avoid chaining